nci-crdc-staging.datacommons.io/manifest.json

```diff
@@ -11,21 +11,21 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2020.09",
-    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2020.09",
-    "datareplicate": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/dcf-dataservice:1.1.1",
-    "fence": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/fence:4.23.5",
+    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2020.10",
+    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2020.10",
+    "datareplicate": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/dcf-dataservice:2020.10",
```

Comment:
nice, is this part of the monthly release now?

Also i'm guessing no changes between 1.11 and 2020.10? cc @BinamB

Reply:
If i remember correctly we added that so that the automation doesnt remove it during code promotion from staging to prod but @themarcelor do we know if this image exists? `707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/dcf-dataservice:2020.10` I've only seen semantic versions for data replication.

Reply:
Also checked that this was in place.